src/js/grapl-cdk/lib/grapl-cdk-stack.ts

```diff
@@ -516,18 +520,17 @@ export class DGraphSwarmCluster extends cdk.NestedStack {
     ) {
         super(parent, id);
 
-        this.dgraphAlphaZone = new route53.PrivateHostedZone(
-            this,
-            'DGraphSwarmZone',
-            {
-                vpc: props.vpc,
-                zoneName: props.prefix.toLowerCase() + '.dgraph.grapl',
-            }
-        );
-
-        this.dgraphSwarmCluster = new Swarm(this, 'DGraphSwarmCluster', {
+        this.dgraphSwarmCluster = new Swarm(this, 'SwarmCluster', {
             prefix: props.prefix,
+            version: props.version,
             vpc: props.vpc,
+            logsGroupResourceArn: super.formatArn({
+                partition: 'aws',
+                service: 'logs',
+                resource: 'log-group',
+                sep: ':',
+                resourceName: `${props.prefix}-grapl-dgraph`
```

Comment:
one thing to check:
when i was dealing with sagemaker arn stuff, I actually had to lower-case-ify the resourceName; you may want to check if the same restriction/expectation is the case here (in case the prefix has upper case). I noticed you have to lowercase the bucketname as well 🤔

Reply:
fixed